### DISCUSSION TITLE
Lesson 1 beginners , SimpleStorage , contract

### DISCUSSION THREAD

**Discussion by Sophalsun888** [smartcontractkit/full-blockchain-solidity-course-py#671] [Q&A] [answered]
Hello 

I can not find how to attach a contract as seen in 1:41:37 of the video .
Do I have to connect my Metamask wallet and have some small amount of eth ? 
![solidity3](https://user-images.githubusercontent.com/63102582/147850818-c49472b2-0500-4b7e-ac24-b6fe0518e18c.png)

**Comment by cromewar** [comment, marked as answer]
Hello @Sophalsun888 you have to "Ctrl + S" in order to compile the contract first.
Also try deleting the cache of your browser and trying again (copy the code before you do this).

**Reply by Sophalsun888** [reply]
thanks :)

**Reply by Yaner52** [reply]
Thank you CromeWar that helped me as well.

**Reply by shucks18** [reply]
Appreciate it !
